app/javascript/packages/document-capture/components/acuant-camera.jsx

```diff
@@ -173,6 +174,24 @@ export const AcuantUIState = {
  * @prop {import('react').ReactNode} children Element to render while camera is active.
  */
 
+/**
+ * Returns a found AcuantCameraUI
+ * object, if one is available.
+ * This function normalizes differences between
+ * the 11.5.0 and 11.7.0 SDKs. The former attached
+ * the object to the global window, while the latter
+ * sets the object in the global (but non-window)
+ * scope.
+ */
+const getActualAcuantCameraUI = () => {
+  if (window.AcuantCameraUI) {
+    return window.AcuantCameraUI;
+  }
+  if (AcuantCameraUI !== 'undefined') {
```

Comment:
This should be a `typeof` check to avoid the thrown error on undeclared variable.

```suggestion
  if (typeof AcuantCameraUI !== 'undefined') {
```